dsc/src/util.rs

```diff
@@ -268,46 +291,92 @@ pub fn write_output(json: &str, format: &Option<OutputFormat>) {
     }
 }
 
-pub fn enable_tracing(trace_level: &Option<TraceLevel>, trace_format: &TraceFormat) {
-    let tracing_level = match trace_level {
-        Some(level) => level,
-        None => {
-            // use DSC_TRACE_LEVEL env var if set
-            match env::var(DSC_TRACE_LEVEL) {
-                Ok(level) => {
-                    match level.to_ascii_uppercase().as_str() {
-                        "ERROR" => &TraceLevel::Error,
-                        "WARN" => &TraceLevel::Warn,
-                        "INFO" => &TraceLevel::Info,
-                        "DEBUG" => &TraceLevel::Debug,
-                        "TRACE" => &TraceLevel::Trace,
-                        _ => {
-                            warn!("Invalid DSC_TRACE_LEVEL value '{level}', defaulting to 'warn'");
-                            &TraceLevel::Warn
-                        },
-                    }
+#[allow(clippy::too_many_lines)]
+pub fn enable_tracing(trace_level_arg: &Option<TraceLevel>, trace_format_arg: &Option<TraceFormat>) {
+    
+    let mut policy_is_used = false;
+    let mut tracing_setting = TracingSetting::default();
+
+    let default_filter = EnvFilter::try_from_default_env()
+        .or_else(|_| EnvFilter::try_new("warning"))
```

Comment:
I think it's `warn` and not `warning`

Reply:
It was a copy-paste from existing code; 
Updated in both places.